netplugin/plugin/netplugin.go

```diff
@@ -220,3 +196,43 @@ func (p *NetPlugin) GetEndpointStats() ([]byte, error) {
 func (p *NetPlugin) InspectState() ([]byte, error) {
 	return p.NetworkDriver.InspectState()
 }
+
+//GlobalFwdModeUpdate update the forwarding mode
+func (p *NetPlugin) GlobalFwdModeUpdate(cfg Config) {
+	var err error
+
+	if p.NetworkDriver != nil {
+		p.NetworkDriver.Deinit()
+		p.NetworkDriver = nil
+	}
+
+	cfg.Instance.StateDriver, _ = utils.GetStateDriver()
+	p.NetworkDriver, err = utils.NewNetworkDriver(cfg.Drivers.Network, &cfg.Instance)
+
+	if err != nil {
+		logrus.Errorf("THE ERROR IS %v", err)
+		return
+	}
+
+	defer func() {
+		if err != nil {
+			p.NetworkDriver.Deinit()
+		}
+	}()
+
+	return
+}
+
+//GetFwdMode returns the fabric forwarding mode
+func GetFwdMode(stateDriver core.StateDriver) string {
+
+	gCfg := mastercfg.GlobConfig{}
+	gCfg.StateDriver = stateDriver
+	err := gCfg.Read("")
+	if err != nil {
+		core.Errorf("Error reading forwarding mode from cluster store")
+		return ""
```

Comment:
@jojimt this will handle the case where netplugin starts first followed by netmaster. In this case global config will not be there during netplugin init. So we default  to bridge mode.